test/suites/dev/moonbase/test-moonbeam-lazy-migrations/test-unlock-democracy-funds.ts

```diff
@@ -0,0 +1,1638 @@
+          "9d045317224c3bb84af1a403f70bd3589ef56",
+        data:
+          "0x00042e0000000080000064a7b3b6e00d00000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207d1c85114953d043c8" +
+          "e741c0a2cb4b8d5cbb1ead482cfdf87fdd66f",
+        data:
+          "0x000c530000000081000010632d5ec76b0500000000000000540000000081000010632d5ec76b05000000" +
+          "00000000550000000081000010632d5ec76b05000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207d7e36637db6a6b1d8" +
+          "308ba3086c4b3108c7de9bbb93a53d7372b9c",
+        data:
+          "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000" +
+          "00000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207e1c1ca61299d69383" +
+          "7af146dddf3f0b8acbd8ca716786db89764ee",
+        data:
+          "0x0004310000000081000064a7b3b6e00d00000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207eacf014ff024ce823" +
+          "c3cdbdf033b0cc3153eaac3cf45882279f153",
+        data:
+          "0x00045600000000860000b2d3595bf00600000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207ee8744bbc02185a04" +
+          "6aa316a897670967fe6105e6258a1a933599f",
+        data:
+          "0x0004540000000080000064a7b3b6e00d00000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207ef3c8caa332c3e602" +
+          "604c910870a7c5c189f6c0ab60d6f13c5880d",
+        data:
+          "0x00043300000000000000f4448291634500000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207f018e75b92d78ef4d" +
+          "e4067706feb9d81518812d41a1542b3c36bc5",
+        data:
+          "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000" +
+          "00000000000000000000000000",
+      },
+    ];
```

Comment:
Please move theses raw storage definitions to a json file or generate them via a ts script

Reply:
This is data scrapped from Moonbeam and I'd like to keep this one. I'll move it in a json file.
I suppose the right place is next to `.ts`.

Reply:
@timbrinded doi you have a strong opinion on where to put this json file?

Reply:
I might have to do it a `.ts` file in order to please the max line length requirement.
Unless there is another doing it?

Reply:
@librelois not with the tests but with the other test data files in `/helpers`

Another example of this would be `/helpers/modexp.ts` which gives the expected results for modexp precompile testing